src/test/java/ee/carlrobert/codegpt/credentials/CredentialsManagerTest.java

```diff
@@ -0,0 +1,104 @@
+package ee.carlrobert.codegpt.credentials;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static testsupport.TestUtil.assertCredentials;
+import static testsupport.TestUtil.assertPassword;
+
+import com.intellij.credentialStore.CredentialAttributes;
+import com.intellij.credentialStore.Credentials;
+import ee.carlrobert.codegpt.TestPasswordSafe;
+import ee.carlrobert.codegpt.credentials.manager.AzureCredentialsManager;
+import ee.carlrobert.codegpt.credentials.manager.LlamaLocalCredentialsManager;
+import ee.carlrobert.codegpt.credentials.manager.LlamaRemoteCredentialsManager;
+import ee.carlrobert.codegpt.credentials.manager.OpenAICredentialsManager;
+import ee.carlrobert.codegpt.credentials.manager.YouCredentialsManager;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CredentialsManagerTest {
+
+  private TestPasswordSafe passwordSafe;
+  private CredentialsService credentialsService;
+
+  @Before
+  public void setUp() {
+    passwordSafe = new TestPasswordSafe();
+    credentialsService = new CredentialsService(passwordSafe);
+  }
+
+  @Test
+  public void testLlamaCredentialsManager() {
```

Comment:
Same thing here, please apply AAA pattern. 
Also, the method name is confusing, can't really tell what it's actually testing without reading  the implementation.

Reply:
Done